extension/encoding/macosunifiedloggingencodingextension/go.mod

```diff
@@ -0,0 +1,49 @@
+module github.com/open-telemetry/opentelemetry-collector-contrib/extension/encoding/macosunifiedloggingencodingextension
```

Comment:
I feel like this is the incorrect module for the go.mod it should be with the BDOT repository.